internal/incident/restore_incidents_test.go

```diff
@@ -0,0 +1,223 @@
+package incident
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"github.com/creasty/defaults"
+	"github.com/icinga/icinga-notifications/internal/config"
+	"github.com/icinga/icinga-notifications/internal/event"
+	"github.com/icinga/icinga-notifications/internal/object"
+	baseConfig "github.com/icinga/icingadb/pkg/config"
+	"github.com/icinga/icingadb/pkg/icingadb"
+	"github.com/icinga/icingadb/pkg/logging"
+	"github.com/icinga/icingadb/pkg/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestIncidentLoading(t *testing.T) {
```

Comment:
This basically is a test for the `LoadOpenIncidents()` function, therefor, please name it `TestLoadOpenIncidents()` and also simply name this file `incidents_test.go` so that it corresponds to the file name the tested function is in.